nowcasting_dataset/data_sources/pv/live.py

```diff
@@ -45,23 +45,42 @@ def get_metadata_from_database() -> pd.DataFrame:
     return pv_systems_df
 
 
-def get_pv_power_from_database(history_duration: timedelta) -> pd.DataFrame:
+def get_pv_power_from_database(
+    history_duration: timedelta, interpolate_minutes: int = 30, load_extra_minutes: int = 60
```

Comment:
I'd recommend settings defaults in exactly one place: either the `model.py` or here. In the past, I've found it slightly hard to debug code when defaults get set in multiple places :slightly_smiling_face:

Reply:
ok, I can remove them here